tendrl/node_agent/objects/volume_alert_counters/__init__.py

```diff
@@ -0,0 +1,24 @@
+from tendrl.commons import objects
+
+
+class VolumeAlertCounters(objects.BaseObject):
```

Comment:
I am somehow not able to convince myself with usage of sds-specific terminologies in a non sds module like node-agent unless absolutely necessary as in case of sds-detection etc..
Can this be achieved by some other means?

Reply:
am i move this to Commons because node-agent have responsible to find all alert counts so it should be accessible by node-agent

Reply:
No don't move this to commons..

Reply:
@r0h4n need your suggestion

Reply:
As suggested by @r0h4n earlier move them under integrations and use